pkg/gen/protobuf_srcs.bzl

```diff
@@ -0,0 +1,64 @@
+# Generated by bazel-generate.sh
+
+protobuf_srcs = [
```

Comment:
Nit: I would expect this variable name to be in `ALL_CAPS`.

Reply:
Done.